diff --git a/vendor/magento/module-inventory-sales/Plugin/Sales/OrderManagement/AppendReservationsAfterOrderPlacementPlugin.php b/vendor/magento/module-inventory-sales/Plugin/Sales/OrderManagement/AppendReservationsAfterOrderPlacementPlugin.php
--- a/vendor/magento/module-inventory-sales/Plugin/Sales/OrderManagement/AppendReservationsAfterOrderPlacementPlugin.php	(date 1640730469081)
+++ b/vendor/magento/module-inventory-sales/Plugin/Sales/OrderManagement/AppendReservationsAfterOrderPlacementPlugin.php	(date 1640730469081)
@@ -193,6 +193,9 @@

         try {
             $order = $proceed($order);
+            if ($order === null) {
+                throw new \RuntimeException('Failed to place order. This is probably due to a payment issue.');
+            }
         } catch (\Exception $e) {
             //add compensation
             foreach ($itemsToSell as $item) {
@@ -203,7 +206,7 @@
             $salesEvent = $this->salesEventFactory->create([
                 'type' => SalesEventInterface::EVENT_ORDER_PLACE_FAILED,
                 'objectType' => SalesEventInterface::OBJECT_TYPE_ORDER,
-                'objectId' => (string)$order->getEntityId()
+                'objectId' => $order ? (string)$order->getEntityId() : '',
             ]);
             $salesEvent->setExtensionAttributes($salesEventExtension);

@@ -211,6 +214,7 @@

             throw $e;
         }
+
         return $order;
     }
 }
