diff --git a/vendor/stripe/module-payments/Observer/WebhooksObserver.php b/vendor/stripe/module-payments/Observer/WebhooksObserver.php
--- a/vendor/stripe/module-payments/Observer/WebhooksObserver.php
+++ b/vendor/stripe/module-payments/Observer/WebhooksObserver.php	(date 1643930237519)
@@ -585,13 +585,18 @@
                 }
                 else
                 {
-                    $state = \Magento\Sales\Model\Order::STATE_PROCESSING;
-                    $status = $order->getConfig()->getStateDefaultStatus($state);
+                    /** @var \Coke\CancelOrder\Helper\Config $cancelOrderConfig */
+                    $cancelOrderConfig = \Magento\Framework\App\ObjectManager::getInstance()
+                        ->create(\Coke\CancelOrder\Helper\Config::class);
+                    if (!$cancelOrderConfig->isEnabled($order->getStoreId())) {
+                        $state = \Magento\Sales\Model\Order::STATE_PROCESSING;
+                        $status = $order->getConfig()->getStateDefaultStatus($state);

-                    $order->setState($state)
-                        ->addStatusToHistory($status, $comment, $isCustomerNotified = false);
+                        $order->setState($state)
+                            ->addStatusToHistory($status, $comment, $isCustomerNotified = false);

-                    $this->orderRepository->save($order);
+                        $this->orderRepository->save($order);
+                    }

                     if ($this->config->isStripeRadarEnabled() && !empty($object['outcome']['type']) && $object['outcome']['type'] == "manual_review")
                         $this->paymentsHelper->holdOrder($order)->save();

