diff --git a/vendor/stripe/module-payments/Helper/Webhooks.php b/vendor/stripe/module-payments/Helper/Webhooks.php
--- a/vendor/stripe/module-payments/Helper/Webhooks.php	(date 1643332850939)
+++ b/vendor/stripe/module-payments/Helper/Webhooks.php	(date 1643332850939)
@@ -79,6 +79,11 @@
             // Retrieve the request's body and parse it as JSON
             $body = $this->request->getContent();

+            try {
+                \Magento\Framework\App\ObjectManager::getInstance()->create(\Coke\CancelOrder\Logger\Logger::class)
+                    ->info('Got a webhook ' . $body);
+            } catch (\Exception $e) { }
+
             $event = json_decode($body, true);
             $stdEvent = json_decode($body);

